examples/text_2_cypher_search.py

```diff
@@ -1,6 +1,6 @@
 from langchain_openai import OpenAI
 from neo4j import GraphDatabase
-from neo4j_genai.retrievers.text_to_cypher import TextToCypherRetriever
+from neo4j_genai.retrievers.text_2_cypher import Text2CypherRetriever
```

Comment:
Should we remove underscores to keep it consistent?
```suggestion
from neo4j_genai.retrievers.text2cypher import Text2CypherRetriever
```

Reply:
Done! I've renamed text_2_cypher text2cypher everywhere to make it consistent across the package